integration/got-ocr2.md

```diff
@@ -0,0 +1,30 @@
+## Extension Manager and GOT-OCR2.0 Loader
+
+## Key Features
+
+### 1. **GOCR2 as Image Reader**
+
+- **GOCR2ImageReader** is a new class designed to read images using the [**GOCR-2.0** OCR engine](https://github.com/Ucas-HaoranWei/GOT-OCR2.0).
+- This reader is initialized with an endpoint that defaults to `http://localhost:8881/ai/infer/` for the OCR service, but can be configured through an environment variable `GOCR2_ENDPOINT` or passed explicitly.
+- It uses exponential backoff retry mechanisms to ensure robustness during API calls.
+- Supports loading image files and extracting their text content, returning structured document data.
+
+#### Setup
+
+- We provide the docker image, with fastapi for serving the GOT-OCR2.0. Pull the image from:
+
+```bash
+docker run -d --gpus all -p 8881:8881 ghcr.io/phv2312/got-ocr2.0:main
```

Comment:
Should we move the docker to Kotaemon ghcr if we use GOCR as the main OCR loader?